autosubmit/scripts/autosubmit.py

```diff
@@ -19,16 +19,15 @@
 
 """Script for handling experiment monitoring"""
 import argparse
-import os
 import sys
 from typing import Optional
-
 from contextlib import suppress
-script_dir = os.path.abspath(os.path.dirname(sys.argv[0]))
-sys.path.append(script_dir)
-sys.path.append(os.path.normpath(os.path.join(script_dir, os.pardir)))
+
 # noinspection PyUnresolvedReferences
 from log.log import Log, AutosubmitCritical, AutosubmitError  # noqa: E402
+import sys
+import os
```

Comment:
This import can be removed :grimacing: But not required for this PR, we can fix it later.